src/components/Form.js

```diff
@@ -1,17 +1,19 @@
 import React, { useState } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { add } from '../redux/books/booksSlice';
+import { useDispatch } from 'react-redux';
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
```

Comment:
- [ ] KIndly solve linters errors instead of disabling them, as they can help you prevent future bugs.